app/components-react/windows/advanced-audio/SourceSettings.tsx

```diff
@@ -49,19 +49,26 @@ export default function AdvancedAudio() {
 }
 
 function PanelHeader(p: { source: AudioSource }) {
-  const { EditorCommandsService, SettingsService } = Services;
+  const { EditorCommandsService, SettingsService, AudioService } = Services;
 
-  const { isAdvancedOutput, recordingTracks, streamTrack, vodTrackEnabled, vodTrack } = useVuex(
-    () => ({
-      isAdvancedOutput: SettingsService.views.isAdvancedOutput,
-      streamTrack: SettingsService.views.streamTrack,
-      recordingTracks: SettingsService.views.recordingTracks,
-      vodTrackEnabled: SettingsService.views.vodTrackEnabled,
-      vodTrack: SettingsService.views.vodTrack,
-    }),
-  );
+  const {
+    isAdvancedOutput,
+    recordingTracks,
+    streamTrack,
+    vodTrackEnabled,
+    vodTrack,
+    muted,
+  } = useVuex(() => ({
+    isAdvancedOutput: SettingsService.views.isAdvancedOutput,
+    streamTrack: SettingsService.views.streamTrack,
+    recordingTracks: SettingsService.views.recordingTracks,
+    vodTrackEnabled: SettingsService.views.vodTrackEnabled,
+    vodTrack: SettingsService.views.vodTrack,
+    // Hack to make muted property reactive
+    muted: AudioService.views.getSource(p.source.sourceId).muted,
```

Comment:
Would it be possible to just try `muted: p.source.muted`.  I don't believe re-creating the source is necessary here.  Just accessing the property within `useVuex` should be enough.